config/crd/bases/rollouts.kruise.io_rollouts.yaml

```diff
@@ -271,6 +275,7 @@ spec:
                     type: boolean
                 type: object
             required:
+            - disabled
```

Comment:
can disabled be optional ? defaults to false?

Reply:
Have already fixed the issues mentioned above